### PR TITLE
CP-12201

### PR DESCRIPTION
**Pull request by jonludlam**
Move some files from dom0.hg into xapi.

### CODE REVIEW

**Comment by jonludlam** [comment]
@xen-git retest this please

File: xapi.spec.in [hunk 1]
```diff
@@ -311,6 +319,9 @@ rm -rf $RPM_BUILD_ROOT
 @OPTDIR@/debug/with-vdi
 %{_unitdir}/cdrommon@.service
 
+
```
Comment (euanh) [inline xapi.spec.in]: Eh? :)

**Comment by euanh** [comment]
@xen-git: Retest this please

File: xapi.spec.in (hunk 1 above)
Comment (jonludlam) [inline xapi.spec.in]: ahem.. fixed :-)

**Comment by jonludlam** [comment]
Don't merge just yet - we'll need to fix up dom0.hg.

**Comment by euanh** [comment]
Looks good to me.   I'll let you orchestrate the required dom0 change.

**Pull request merged**